src/riak_core_membership_leave.erl

```diff
@@ -129,7 +129,7 @@ check_balanced_or_wanting(Ring) ->
 %% although this may result in a large number of transfers
 -spec attempt_simple_transfer(transfer_ring(),
                                 term(),
-                                random:ran(),
```

Comment:
Both random:rand and rand:state are undocumented functions... Since this is legacy code it might be fine. In theory, one should possibly use the rand library different in the solution.

Reply:
rand:state() is a documented type, exported by the rand module - https://www.erlang.org/doc/man/rand#type-state.  The function rand:uniform_s/2 has it as both an input and output - https://www.erlang.org/doc/man/rand#uniform_s-2.